src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1272,13 +1279,16 @@ static bool addTriggeredSubscriptions_noCache
       long long           lastNotification   = sub.hasField(CSUB_LASTNOTIFICATION)? getIntOrLongFieldAsLongF(sub, CSUB_LASTNOTIFICATION) : -1;
       std::string         renderFormatString = sub.hasField(CSUB_FORMAT)? getStringFieldF(sub, CSUB_FORMAT) : "";
       RenderFormat        renderFormat       = stringToRenderFormat(renderFormatString);
+      ngsiv2::HttpInfo    httpInfo;
+
+      httpInfo.url = sub.hasField(CSUB_REFERENCE)? getStringFieldF(sub, CSUB_REFERENCE) : "";
 
```

Comment:
I'd should used "FIXME PR" instead of "FIXME" as it has to be fixed before the "master PR" feature/ -> develop get finshed... but ok, it's fine. NTC.